.circleci/config.yml

```
@@ -1,20 +1,61 @@
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:12.9.1
```

Can probably use the latest 

```suggestion
    - image: circleci/node:12
```
... if that's a thing, no?

---

yes, that version is supported: https://circleci.com/docs/2.0/docker-image-tags.json 👍